code/modules/mob/living/carbon/xenomorph/xenoprocs.dm

```diff
@@ -57,9 +47,11 @@
 			if(0 to 0.33)
 				hp_color = "red"
 
+		var/distance = get_dist(user, X)
+
 		xenoinfo += " <b><font color=[hp_color]>Health: ([X.health]/[X.maxHealth])</font></b>"
 
-		xenoinfo += " <b><font color=green>([AREACOORD_NO_Z(X)])</b></td></tr>"
+		xenoinfo += " <b><font color=green>([AREACOORD_NO_Z(X)][distance > 0 ? " <b>Distance: [distance]</b>" : ""])</font></b></td></tr>"
```

Comment:
Why is this here move it to right next to where it's used or even better just inline it

Reply:
I can if you insist, but it's used twice, so it would have to be inlined twice.